combat-trainer.lic

```diff
@@ -38,19 +37,19 @@ class SetupProcess
           break_summoned_weapon?(checkleft)
         end
       else
-        EquipmentManager.instance.stow_weapon(@weapon_training[game_state.weapon_skill])
+        EquipmentManager.instance.stow_weapon(game_state.weapon_training[game_state.weapon_skill])
```

Comment:
This is another one that feels like it should be combined into game_state. `game_state.get_weapon` or something.